### DISCUSSION TITLE
If 2.0.1 works fine on my E3v2, is it still worth to self-compile the newest version?

### DISCUSSION THREAD

**Discussion by alphatrianguli** [Jyers/Marlin#1944] [Q&A]
Noob question here: If my E3v2 works fine on the latest pre-compiled version of Jyers, 2.0.1, is it still worth to compile my own FW on base of the newest branch by [tititopher68-dev](https://github.com/tititopher68-dev) to get new features or really improving bug-fixes?
Many thanks on some recommendations on this.

**Comment by Cal-LF** [comment]
My 2 cents, it is worth it. I have been compiling and installing each update.

**Reply by alphatrianguli** [reply]
Thanks for the reply. I was expecting something like that, allthough it would be nice to know, what the most notable improvements are? :)

**Reply by alphatrianguli** [reply]
@Cal-LF 
Can you quickly tell me, which configuration file you are using? Having issues finding the proper one. I connected my BLTouch directly to the dedicated port (all 5 pins) on my 4.2.2. board and I want to use UBL 10x10. Many thanks!

**Reply by Cal-LF** [reply]
@alphatrianguli I use the manual mesh configs.
You pick the config for your setup. https://github.com/tititopher68-dev2/Marlin/tree/new_feature/Configuration%20Files/Ender3V

[Compiling Example.txt](https://github.com/Jyers/Marlin/files/8597888/Compiling.Example.txt)
[Compiling Steps.txt](https://github.com/Jyers/Marlin/files/8597889/Compiling.Steps.txt)

**Comment by vhristenco** [comment]
Compiled the latest sources today and... those probing feedrates are crazy. Tried to do a manual tramming and at the second point I got a high pitch noise, then the print head quickly crashed into the bed.
